### PR TITLE
YC links upgrade

### PR DESCRIPTION
**Pull request by li-ar**
Add first release version of youcontrol links. 
There are some changes in common code to (new parameters for search results macro) but it should not affect any existing logic

### CODE REVIEW

File: declarations_site/catalog/templates/_youcontrol.jinja [hunk 1]
```diff
@@ -5,7 +5,7 @@
     </svg>
 {% endmacro %}
 
-{% macro yc_check_person(option, class_name = '', size = 'sm') %}
+{% macro yc_check_person(option, class_name = '', size = 'sm', name = '', page = '') %}
     {% if size == 'sm' %}
         {% if option == 'a' %}
         <a href="#" class="{{ class_name }} youcontrol__check-person youcontrol__check-person--option-a youcontrol__check-person--sm" title="{{ _("Досьє від Youcontrol") }}">
```
Comment (dchaplinsky) [inline declarations_site/catalog/templates/_youcontrol.jinja:11]: Links here and below (except for the option B) are left empty (#) is that a bug or it was left as is intentionally?

File: declarations_site/catalog/templates/declaration.jinja [hunk 2]
```diff
@@ -13,7 +13,7 @@
 
 {% block html_desc %}{% if meta %}{{ meta.description }}{% else %}{{ _("Д​екларації") }}{% if declaration.declaration.source == "NACP" %} {{ _("НАЗК") }}{% endif %} {{ _("про майно, доходи, витрати та зобов'язання фінансового характеру") }} {{ declaration.general.last_name }} {{ declaration.general.name }} {{ _("за") }} {{ declaration.intro.declaration_year }} {{ _("рік онлайн у відкритій базі даних проекту") }}{% endif %}{% endblock %}
 
-{% block bodyclass %}not-front {% if branding_link %}branding-page {% endif %}declaration-page {% if declaration.declaration.source == "NACP" %}declaration-page-nacp{% endif %}{% endblock %}
+{% block bodyclass %}not-front body--yc-option-b {% if branding_link %}branding-page {% endif %}declaration-page {% if declaration.declaration.source == "NACP" %}declaration-page-nacp{% endif %}{% endblock %}
```
Comment (dchaplinsky) [inline declarations_site/catalog/templates/declaration.jinja:16]: Initial idea was to test 4 different options, are you intentionally limiting it to option b?

File: declarations_site/catalog/templates/_youcontrol.jinja (hunk 1 above)
Comment (li-ar) [reply declarations_site/catalog/templates/_youcontrol.jinja:11]: It's OK, coz we are testing only option B. But we keep another options which may have different links for the future

File: declarations_site/catalog/templates/declaration.jinja (hunk 2 above)
Comment (li-ar) [reply declarations_site/catalog/templates/declaration.jinja:16]: Yep, we decided to hardcode version B for now

**Pull request merged**